tests/SprykerSdkTest/Spryk/Model/Spryk/Executor/ConditionMatcherTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Model\Spryk\Executor;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Argument;
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollection;
+use SprykerSdk\Spryk\Model\Spryk\Executor\ConditionMatcher\ConditionMatcher;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\ExpressionLanguage\SyntaxError;
+
+class ConditionMatcherTest extends Unit
+{
+    /**
+     * @var \SprykerSdk\Spryk\Model\Spryk\Executor\ConditionMatcher\ConditionMatcher
+     */
+    protected $conditionMatcher;
+
+    /**
+     * @return void
+     */
+    public function setUp(): void
+    {
+        parent::setUp();
```

Comment:
I personally don't like to use this approach. What's against using `\SprykerSdkTest\Module\SprykHelper::getClass()` or adding your own method to get this service from the tester as done with e.g. `\SprykerSdkTest\Module\SprykHelper::getClassDumper()`?